<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pixel</class>
 <widget class="QMainWindow" name="Pixel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pixel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="draw_b_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>181</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="x_one">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="y_one">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Part B - w /  Part A - radius</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="x_two">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="y_two">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_a_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>181</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw A</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
